FogFog-iOS/FogFog-iOS/App/AppDelegate.swift

```diff
@@ -34,5 +38,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         return false
     }
+    
+    // 첫 실행 시 키체인 정보 초기화하는 메서드
+    private func resetKeyChainAtFirstLaunch() {
```

Comment:
p3;
```suggestion
    private func resetKeychainAtFirstLaunch() {
```
키체인 네이밍이 공식문서도 그렇고, 다른 파일에서도 `Keychain`과 같이 사용되고 있어서 통일해주면 좋을 것 같아요!